server/index.js

```diff
@@ -7,12 +8,12 @@ const app = express();
 app.use(express.urlencoded(({ extended: true })));
 app.use(express.static('public'));
 app.use((req, res, next) => {
-  res.header('Access-Control-Allow-Origin', 'http://localhost:1984');
+  res.header('Access-Control-Allow-Origin', 'http://localhost:3000');
   res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
   next();
 });
 
-const PORT = 3000;
+const PORT = process.env.PORT || 3003;
```

Comment:
Was this necessary? I changed it back to use a static port because I thought it might have been causing issues for me when I was trying to get docker running.

Reply:
It wasn't necessary for Docker, but I added it in anticipation of AWS integration.
Early research suggested that services like AWS may use its own port setup and this catches that. If it causes problems when integrating, I'll change it back.